contributing/development/core_and_modules/variant_class.rst

```diff
@@ -6,22 +6,20 @@ Variant class
 About
 -----
 
-Variant is the most important datatype of Godot, it's the most important
-class in the engine. A Variant takes up only 20 bytes and can store
-almost any engine datatype inside of it. Variants are rarely used to
-hold information for long periods of time, instead they are used mainly
-for communication, editing, serialization and generally moving data
-around.
+Variant is the most important datatype in Godot. A Variant takes up only 20
```

Comment:
> only 20 bytes

On arm64 macOS, it's `24` (probably the same on all 64-bit platforms, because of 8 byte alignment).